awx/main/models/credential/__init__.py

```diff
@@ -315,6 +317,15 @@ def _get_dynamic_input(self, field_name):
         else:
             raise ValueError('{} is not a dynamic input field'.format(field_name))
 
+    def validate_role_assignment(self, actor, role_definition):
+        if isinstance(actor, User):
+            if actor.is_superuser or self.organization in actor.organizations:
```

Comment:
`actor.organizations` wasn't correct, and it was my fault. https://github.com/ansible/awx/pull/15298